concent_api/utils/decorators.py

```diff
@@ -43,16 +43,16 @@ def wrapper(request, *args, **kwargs):
                     return JsonResponse({'error': 'Client Authentication message not included'}, status = 400)
             except Http400 as exception:
                 return JsonResponse({'error': f'{exception}'}, status = 400)
-            except FieldError:
-                return JsonResponse({'error': "Golem Message contains wrong fields."}, status = 400)
-            except MessageFromFutureError:
-                return JsonResponse({'error': 'Message timestamp too far in the future.'}, status = 400)
-            except MessageTooOldError:
-                return JsonResponse({'error': 'Message is too old.'}, status = 400)
+            except FieldError as exception:
+                return JsonResponse({'error': f'Golem Message contains wrong fields.{exception}'}, status = 400)
```

Comment:
It would be nice if there was a space between our message and the exception message.

Reply:
Done.

Reply:
Well, that will work but it's a lazy solution ;)

Maybe you could create a very simple utility function that will join N strings, add spaces between them and ignore the empty ones so that there's no separator if not needed?
``` python
join_messages('Golem Message contains wrong fields.', str(exception))
```

I wanted a suggest a simple one-liner but I think it would be too verbose to just paste it everywhere so putting it into a utility function would be better:

``` python
' '.join(m for m in messages if m not in ['', None])
```

Reply:
Okey, I've added join_messages() function, I'm not sure if it's easier to read but maybe we could use it more in the future.